pkg/finality-grandpa/bridge_state.go

```diff
@@ -0,0 +1,101 @@
+// Copyright 2023 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package grandpa
+
+import (
+	"sync"
+)
+
+type waker struct {
+	mtx    sync.RWMutex
```

Comment:
```suggestion
	sync.RWMutex
```
common pattern is to not use a variable name here!